Sage-Bionetworks/schematic
# BugFix: Fixed an issue where files on synapse were not being annotated correctly

Fixed an issue where annotations for files were not being added correctly. 
For file-based metadata, the table ID was being added for the `entityId` column, and this synId was then used as the target for which entity the annotations should be applied to. This PR now stores the entityId of the files themselves so that they can be annotated properly.

## To Test

- Submit [manifest](https://docs.google.com/spreadsheets/d/1q4VaK4ta2WEx4O2u9IztGrn2MJ01Rirc9Ca2OI_FoF4/edit#gid=0) with `-mrt table_and_file` and `-tm replace`
- Check annotations for dummy dataset files in dataset syn39241199
